pipeline/beam_tables.py

```diff
@@ -86,23 +92,22 @@
     'as_traffic': ('integer', 'nullable'),
 """
 
-# Mapping of each scan type to the zone to run its pipeline in.
-# This adds more parallelization when running all pipelines.
-SCAN_TYPES_TO_ZONES = {
-    'https': 'us-east1',  # https has the most data, so it gets the best zone.
-    'http': 'us-east4',
-    'echo': 'us-west1',
-    'discard': 'us-west2',
-    'satellite': 'us-central1'  # us-central1 has 160 shuffle slots by default
-}
 
-ALL_SCAN_TYPES = SCAN_TYPES_TO_ZONES.keys()
+def _add_schemas(schema_a: Dict[str, Any],
+                 schema_b: Dict[str, Any]) -> Dict[str, Any]:
+  """Add two bigquery schemas together."""
+  full_schema: Dict[str, Any] = {}
+  full_schema.update(schema_a)
+  full_schema.update(schema_b)
+  return full_schema
 
-# Data files for the non-Satellite pipelines
-SCAN_FILES = ['results.json']
 
-# An empty json file is 0 bytes when unzipped, but 33 bytes when zipped
-EMPTY_GZIPPED_FILE_SIZE = 33
+HYPERQUACK_BIGQUERY_SCHEMA = _add_schemas(
```

Comment:
Why is this and ADDITIONAL_HYPERQUACK_BIGQUERY_SCHEMA in different files?
Define only the former. The later could be inlined in the _add_schemas call.

Reply:
Moved both schemas back to the beam tables file and inlined them.